sql/migrations/error_detail_reports/000001_create_error_detail_reports.up.sql

```diff
@@ -17,6 +17,9 @@ CREATE TABLE IF NOT EXISTS error_detail_reports (
 		status_code INT,
 		error_message TEXT DEFAULT '',
 		event_type TEXT DEFAULT '',
+		event_name TEXT DEFAULT '',
```

Comment:
These changes should be written (alter table, add columns) in a new migration sql file.

Reply:
Moved